data_structures/src/list/sqlist.c

```diff
@@ -0,0 +1,199 @@
+/**
+ * @file
+ * @brief Implementation of sequential list with basic operations and algorithms
+ * @details
+ * @author danielhu19 (shady030314@gmail.com)
+ * @see  Wikipedia: https://en.wikipedia.org/wiki/List_(abstract_data_type))
```

Comment:
`@see` is only used to link to other files. Please link to Wikipedia/algorithm/implementation explanation in a Markdown link in the brief description section.

Reply:
Can you check my `<forehead.h>` macros comments? Should I use `@def` and remove the `@link`? I'm not so familiar with Doxygen.

Reply:
Sure. I'll check it out once all the reviews are addressed.